app/views/tracks/index.html.haml

```diff
@@ -29,6 +29,13 @@
     .row
       .col-md-12
         %table.table.table-striped#tracks
+          %th
+          %th
+          %th
+          %th From
+          %th To
+          %th In
```

Comment:
Perhaps that's more meaningful if it says 'Room' or 'In room', what do you think?